src/server/utils/cassandra.js

```diff
@@ -29,7 +52,7 @@ export function getCassandraClientAsync(keyspace) {
       }
       
       // Create a client and promisify it
-      let client = new Client(clientOpts);
+      let client = new dse.Client(clientOpts);
```

Comment:
The `dse.` is not necessary here either because `Client` is being imported already. It can just stay `new Client(...)`.